app/models/ability/manager.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
```

Comment:
Managers are broken. Need a discussion with Emma G about best route forward.
Roles should either be global, or specific: 
- You can be an administrator
- You can be an owner *of* a study

Manager is both. And given any registered user of Sequencescape can become a manager by creating a project (which they have permission to to) it is trivial to get these permissions.